src/app/pages/BuyWithCreditCard/BuyWithCreditCard.tsx

```diff
@@ -123,6 +124,10 @@ export const BuyWithCreditCard: FC = () => {
     ? toLocalFormat(inputCurrency.maxAmount, { decimalPlaces: inputCurrency.precision })
     : undefined;
 
+  const outputAmountIsLegit = outputAmount && outputAmount > 0;
+  const submitDisabled =
+    !purchaseLink || isDefined(updateLinkError) || Object.keys(errors).length > 0 || !outputAmountIsLegit;
```

Comment:
It may be better to add `isString` helper from 'templewallet-mobile' and use it here